components/up-job.libsonnet

```diff
@@ -0,0 +1,79 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
+
+{
+  local up = self,
+
+  config:: {
+    name: error 'must provide name',
+    version: error 'must provide version',
+    image: error 'must provide image',
+    backoffLimit: error 'must provide backoffLimit',
+    writeEndpoint: error 'must provide writeEndpoint',
+    readEndpoint: error 'must provide readEndpoint',
+
+    commonLabels:: {
+      'app.kubernetes.io/name': 'observatorium-up',
+      'app.kubernetes.io/instance': up.config.name,
+      'app.kubernetes.io/version': up.config.version,
+      'app.kubernetes.io/component': 'test',
+    },
+  },
+
+  job:
+    local job = k.batch.v1.job;
+    local container = job.mixin.spec.template.spec.containersType;
+
+    local c =
+      container.new('observatorium-up', up.config.image) +
```

Comment:
I am missing some flag for configuring what tests to run.. Is this hardcoded?

Reply:
sorry the git diff makes this hard to follow, this file was just renamed from `up.libsonnet` to `up-job.libsonnet`, separating the job that we use in our e2e tests, and the long running process we run in prod